crates/sargon/src/profile/logic/account/account_visibility.rs

```diff
@@ -0,0 +1,49 @@
+use crate::prelude::*;
+
+impl Account {
+    /// Marks the account as hidden
+    pub fn mark_as_hidden(&mut self) {
+        self.flags.insert_flag(EntityFlag::DeletedByUser);
+    }
+
+    /// Marks the account as tombstoned
+    pub fn mark_as_tombstoned(&mut self) {
+        self.flags.insert_flag(EntityFlag::TombstonedByUser);
```

Comment:
we should probably unhide it before marking it tombstoned. I mean we should avoid `[.deleted, .tombstoned]` we should have only `[.tombstoned]`

Reply:
Wanted to ask about this, thanks for reminding me.